tests/test_transformers.py

```diff
@@ -708,6 +708,26 @@ def test_simple(self):
         assert valid['accuracy'] > 0.97
         assert test['accuracy'] > 0.97
 
+    @testing_utils.skipUnlessGPU
+    def test_eval_classifier(self):
+        valid, _ = testing_utils.eval_model(
+            dict(
+                task='dialogue_safety:adversarial,dialogue_safety:standard',
```

Comment:
nice. can we do only 1 of the tasks to make it faster?

Reply:
uhh this is a very very very very expensive test, please don't put this in main. a nightly test? Or otherwise can we just tack it on to the other classifier test? Because this is going to download a giant file...